docs/docs/userguide/CLEANUP.md

```diff
@@ -7,9 +7,7 @@ This guide describes how to uninstall all Atlassian Data Center products and rem
 
     Before you begin, make sure that you have an up-to-date backup available in a secure location. 
 
-If you want to uninstall just one product, don't proceed with the uninstallation process and remove it from the configuration file instead. For more information, see [Configuration](CONFIGURATION.md). 
```

Comment:
This is removed since we only support Bamboo at the moment. we can add this later